### PR TITLE
CO-924 Add new module for adapting compassion mailbox in v10

### PR DESCRIPTION
**Pull request by ecino**

### CODE REVIEW

File: shared_inbox_switzerland/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
```
Comment (eicher31) [inline shared_inbox_switzerland/__init__.py]: is not \_\_manifest__.py ?

File: shared_inbox_switzerland/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
```
Comment (eicher31) [inline shared_inbox_switzerland/models/__init__.py]: Same

File: shared_inbox_switzerland/models/mail_channel.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
```
Comment (eicher31) [inline shared_inbox_switzerland/models/mail_channel.py]: Same

**Comment by ecino** [comment]
Thanks. I had to change it everywhere :+1:

**Review by eicher31 — APPROVED** [review]

**Pull request merged**